src/test/reporters/json.ts

```diff
@@ -200,9 +201,20 @@ class JSONReporter implements Reporter {
   }
 
   private _serializeTestSpec(test: TestCase): JSONReportSpec {
+
```

Comment:
My comments are mostly nits, not sure why eslint did not complain here:
- remove empty line

Reply:
No problem. I'm not a JS / TS developer, so it's all relatively new to me. I'll update with a new commit shortly